homework_07.py

```diff
@@ -0,0 +1,50 @@
+phone_book = {}
+
+def stats():
+    print("Кількість записів: {}".format(len(phone_book)))
+
+def add():
+    name = input("Введіть ім'я: ")
+    phone_number = input("Введіть номер телефону: ")
+    phone_book[name] = phone_number
+    print("Запис додано")
+
+def delete(name):
+    if name in phone_book:
+        del phone_book[name]
+        print("Запис видалено")
+    else:
+        print("Запис з іменем {} не знайдено".format(name))
+
+def list_names():
+    print("Контакти:")
+    for name in phone_book:
+        print(name)
+
+def show(name):
+    if name in phone_book:
+        print("Ім'я: {}".format(name))
+        print("Номер телефону: {}".format(phone_book[name]))
+    else:
+        print("Запис з іменем {} не знайдено".format(name))
+
+
+while True:
+    print("\n Введіть команду: Контакти, Додати, Видалити <name>, Список, Показати <name>, або Вихід")
+    command = input()
+    if command == "Контакти":
```

Comment:
nit: better to use English words in code in order for other developers could understand it